test-dotnet/action.yml

```diff
@@ -21,47 +21,29 @@ inputs:
   dotnetVersion:
     description: "Version of dotnet to use. Default is v7.x."
     required: false
-    type: string
     default: "7.x"
+  actionsRepoRef:
+    description: "Run actions from this ref. Default is master."
+    required: false
+    default: "master"
 
 runs:
   using: "composite"
   steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - name: .NET Install
-      env:
-        DOTNET_INSTALL_DIR: "./.dotnet"  # Required by the runner to install dotnet
-      uses: actions/setup-dotnet@v3
+  
+    - name: Checkout trakx/github-actions repo
+      uses: actions/checkout@v4
       with:
-        dotnet-version: ${{ inputs.dotnetVersion }}
+        repository: trakx/github-actions
+        path: ./github-actions-build-push-container-image
```

Comment:
I'd suggest to pick a path that is "more less" the current action name.
In this case would be something like:
```
path: ./github-actions-test-dotnet
```

This avoid to have conflicts when we are reusing actions. At least I've struggled a lot due to this when creating shared workflows 😞

Reply:
It is a mistake, thanks for spotting it